pkg/dispatchcli/cmd/install_config.go

```diff
@@ -54,10 +54,14 @@ openfaas:
 kafka:
   chart:
     chart: kafka
-    namespace: riff-system
+    namespace: dispatch
```

Comment:
Should we just default kong and postgres to also deploy into the dispatch namespace?

Reply:
Yes, I think it's time to make the single namespace the default deployment model, I'll open an issue.